streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/canopy_cover_edit.html

```diff
@@ -65,253 +62,247 @@
 </style>
 
 {% if user.is_authenticated %}
-    {% if added %}
-        <strong>{% trans "You have successfully submitted your Canopy Cover Survey." %}</strong>
-
-    {% else %}
-    <h3 align="center">
-        <a href="{% url 'streamwebs:site' site.site_slug %}">
-            {{ site.site_name }}
-        </a>
-    </h3>
-    <h4 align="center">{% trans 'New Canopy Cover Sampling' %}</h4>
-    <form id='canopy_cover_form' method='post'>
-    {% csrf_token %}
-        <table> <!-- main cc info table -->
-            <tr>{{ canopy_cover_form.non_field_errors }}</tr>
-            <tr>
-                <th align='left'>{{ canopy_cover_form.school.label }}</th>
-                <td>{{ canopy_cover_form.school }}</td>
-                <td>{{ canopy_cover_form.school.errors }}</td>
-            </tr>
-            <tr>
-                <th align='left'>{{ canopy_cover_form.date_time.label }}</th>
-                <td>{{ canopy_cover_form.date_time }}</td>
-                <td>{{ canopy_cover_form.date_time.errors }}</td>
-            </tr>
-            <tr>
-                <th align='left'>{{ canopy_cover_form.weather.label }}</th>
-                <td>{{ canopy_cover_form.weather }}</td>
-                <td>{{ canopy_cover_form.weather.errors }}</td>
-            </tr>
-            <tr style='display: none'>
```

Comment:
You need to keep these inputs, because these are what are used to actually send the data to the server about the CC data. Without them it just gives you "This field is required" errors when you try to submit the form.